code/modules/research/designs/weapon_designs.dm

```diff
@@ -234,13 +234,13 @@
 	category = list("Firing Pins")
 	departmental_flags = DEPARTMENTAL_FLAG_SECURITY
 
-/datum/design/pin_away
-	name = "Station Locked Pin"
-	desc = "This is a security firing pin which only authorizes users who are off station."
-	id = "pin_away"
+/datum/design/pin_explorer
+	name = "Outback Firing Pin"
+	desc = "This firing pin only shoots while ya ain't on station, fair dinkum!"
```

Comment:
This really isnt sci-y

Reply:
:koala: Whatever you say, fella. :kangaroo: